pom.xml

```diff
@@ -53,6 +53,12 @@ all modules and does not describe the project structure per-subproject.
     <version>1.3.2</version>
     <scope>compile</scope>
   </dependency>
+  <dependency>
+    <groupId>org.jetbrains.dokka</groupId>
+    <artifactId>dokka-base</artifactId>
+    <version>1.5.0</version>
```

Comment:
As discussed previously, we need to grab the latest updates from `config`. So that here we would have the latest version.

Reply:
Yes, I remember that. I have pulled the config and am in the process of cleaning unresolved references. I will generate the pom and the license report again later.